python/sqlflow/column/tensorflow.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow import feature_column as tfc
+
+
+def numeric(field, dim=0):
```

Comment:
I feel very difficult to understand the Python code here. Python doesn't allow programmers to declare parameter types and return types. I have no idea what `field` is without an exhaustively detailed comment.  If we are going to have a large Python codebase, we must follow the code quality of Google Tangent.  For example: https://github.com/google/tangent/blob/master/tangent/compile.py